Vehicle.cpp

```diff
@@ -0,0 +1,22 @@
+
+#include "Vehicle.h"
+
+// Vehicle::Vehicle(const std::string& n) : name(n) { }
+    
+Vehicle::~Vehicle() = default;
+Vehicle::Vehicle(const Vehicle&) = default;
+// Vehicle& operator= (const Vehicle&);
```

Comment:
Don't declare a special member function and then fail to implement it.
Implement it using `= default;`
- [ ] revise